File: _scratch/ScottStuff/en-us/channel-inspector/files/GroupMe/SigninCard/example1.cs
public async Task ExecuteCommand(ActivityContext activityContext, Activity activity)
{
	IMessageActivity iMessage = (IMessageActivity)activity;
	var reply = activity.CreateReply("");
	reply.Summary = $"Request to sign in";
	reply.Attachments = new List<Attachment>();
	reply.Attachments.Add(new Attachment()
	{
		ContentType = SigninCard.ContentType,
		Content = new SigninCard()
		{
			Text = "Login to signin sample",
			Buttons = new List<CardAction>()
			{
				new CardAction()
				{
					Title = "Signin",
					Type = ActionTypes.Signin,
					Value = "https://login.live.com/login.srf?wa=wsignin1.0&ct=1464753247&rver=6.6.6556.0&wp=MBI_SSL&wreply=https:%2F%2Foutlook.live.com%2Fowa%2F&id=292841&CBCXT=out"
				}
			}
		}
	});

	await SendResponse(activityContext, reply).DropContext();
}

File: _scratch/ScottStuff/en-us/channel-inspector/files/Slack/SigninCard/example1.js
function BotSendSignIn(session, builder){
	var signin_card = new builder.SigninCard(session)
	.text("Login to signin sample")
	.button("Signin", "https://login.live.com/login.srf?wa=wsignin1.0&ct=1464753247&rver=6.6.6556.0&wp=MBI_SSL&wreply=https:%2F%2Foutlook.live.com%2Fowa%2F&id=292841&CBCXT=out");

	var msg = new builder.Message(session)
	.attachments([signin_card]);

	session.send(msg);
}
